### PR TITLE
Add cards state

### PR DESCRIPTION
**Pull request by dmitryRomankov**

### CODE REVIEW

File: client/src/store/cards/cards.effects.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Actions, Effect, ofType } from '@ngrx/effects';
+import { of, Observable } from 'rxjs';
+import { catchError, map, switchMap } from 'rxjs/operators';
+import { Action } from '@ngrx/store';
+
+import * as cardActions from './cards.action';
+import { Card } from 'models';
+@Injectable()
```
Comment (vharadkou) [inline client/src/store/cards/cards.effects.ts]: empty space

File: client/src/store/cards/cards.effects.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+export class CardsEffects {
+
+  @Effect() public getCards$: Observable<Action> = this.actions$.pipe(
+    ofType<cardActions.GetCards>(cardActions.CardsActionTypes.GetCards),
+    switchMap((action: cardActions.GetCards) =>
+      this.http.get('http://www.mocky.io/v2/5bd6f6183500000523fd7e40').pipe(
+        map((data: Card[]) => new cardActions.GetCardsSuccess(data)),
+        catchError((error) => of(new cardActions.GetCardsError(error))
+        ))
```
Comment (vharadkou) [inline client/src/store/cards/cards.effects.ts]: formatting

File: client/src/store/cards/index.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+export * from './cards.action';
+// export * from './cards.effects';
```
Comment (vharadkou) [inline client/src/store/cards/index.ts]: remove

File: client/tslint.json [hunk 4]
```diff
@@ -1,101 +1,133 @@
 {
-  "rulesDirectory": [
```
Comment (vharadkou) [inline client/tslint.json]: please don't remove codelyzer

File: client/src/store/cards/cards.effects.ts (hunk 1 above)
Comment (dmitryRomankov) [reply client/src/store/cards/cards.effects.ts]: Angular code style recommends separate imports from third party and local files

**Pull request merged**